components/WhyMeditate/Captions.module.css

```diff
@@ -0,0 +1,45 @@
+.word {
+  font-size: 8px;
+}
+.word.isSpecial {
+  font-style: italic;
+  fill: #a85a5d;
+}
+
+@keyframes appear {
+  0% {
+    color: #a85a5d;
+    border-radius: 10px;
+  }
+  20% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 1;
+    background: transparent;
+  }
+}
```

Comment:
The 0% changes the color and border radius, but the 100% changes the bg and opacity. They should agree on changing the same things.

Reply:
I have deleted the animation, since we are using SVG text.